docs/install/containers/images/backend.md

```diff
@@ -39,13 +39,13 @@ We encourage users of the `Plone` images to familiarize themselves with the opti
 
 ### Main variables
 
-| Environment variable | Zope option | Default value |
-| --- | --- | --- |
-| `DEBUG_MODE` | `debug-mode` | `off` |
-| `SECURITY_POLICY_IMPLEMENTATION` | `security-policy-implementation` | `C` |
-| `VERBOSE_SECURITY` | `verbose-security` | `false` |
-| `DEFAULT_ZPUBLISHER_ENCODING` | `default-zpublisher-encoding` | `utf-8` |
-| `LISTEN_PORT` | (no equivalent) | `8080` |
+| Environment variable             | Zope option                      | Default value |
+| -------------------------------- | -------------------------------- | ------------- |
+| `DEBUG_MODE`                     | `debug-mode`                     | `off`         |
+| `SECURITY_POLICY_IMPLEMENTATION` | `security-policy-implementation` | `C`           |
+| `VERBOSE_SECURITY`               | `verbose-security`               | `false`       |
+| `DEFAULT_ZPUBLISHER_ENCODING`    | `default-zpublisher-encoding`    | `utf-8`       |
+| `LISTEN_PORT`                    | (no equivalent)                  | `8080`        |
```

Comment:
Please configure your editor to not reformat tables.

Reply:
Here is a before and after. 
I personally prefer the (editor extension) reformatted version.


![Item-0_and_AppleTimeMachineExtra_and_Item-0_and_Item-0](https://user-images.githubusercontent.com/1144737/208406869-05bed620-9f57-49ae-b6f1-e3a33de4c112.png)
 
![Item-0_and_AppleTimeMachineExtra_and_Item-0_and_Item-0](https://user-images.githubusercontent.com/1144737/208407091-13a922d3-4b0b-4743-b376-011ee01563ac.png)

Reply:
Please not in this PR. It just adds noise, and we need to discuss whether to reformat all tables in the Docs Team. I find it easier to read diffs when they have shorter lines, and the reformat makes it exceedingly difficult to do that.